src/components/InputTodo.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+
+class InputTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      title: '',
+    };
+  }
+
+  onChange = (e) => {
+    this.setState({
+      title: e.target.value,
+    });
+  };
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+  }
```

Comment:
To make the form functional, you should update the handleSubmit function to call a function passed down via props that will handle the submission of the form data.